client/state/data-layer/wpcom/jetpack-onboarding/test/index.js

```diff
@@ -5,13 +5,111 @@
  */
 import { http } from 'state/data-layer/wpcom-http/actions';
 import {
+	requestJetpackOnboardingSettings,
 	saveJetpackOnboardingSettings,
 	storeJetpackOnboardingSettings,
+	announceRequestFailure,
 	announceSaveFailure,
 } from '../';
 import { JETPACK_ONBOARDING_SETTINGS_SAVE } from 'state/action-types';
 import { updateJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
 
+describe( 'announceRequestFailure()', () => {
+	const dispatch = jest.fn();
+
+	test( 'should trigger an error notice upon unsuccessful save request', () => {
+		announceRequestFailure( { dispatch } );
+
+		expect( dispatch ).toHaveBeenCalledWith(
+			expect.objectContaining( {
+				notice: expect.objectContaining( {
+					status: 'is-error',
+					text: 'Could not fetch settings from site. Please try again later.',
+				} ),
+			} )
+		);
+	} );
+} );
+
+describe( 'requestJetpackOnboardingSettings()', () => {
+	const dispatch = jest.fn();
+	const token = 'abcd1234';
+	const userEmail = 'example@yourgroovydomain.com';
+	const siteId = 12345678;
+
+	const action = {
+		type: JETPACK_ONBOARDING_SETTINGS_SAVE,
```

Comment:
Did you mean `JETPACK_ONBOARDING_SETTINGS_REQUEST`?

Reply:
No, that's what https://github.com/Automattic/wp-calypso/pull/21013/files#diff-d6e26b3c448fc82f9820bc5db48730d4R57 dispatches. However, _this_ line should be `JETPACK_ONBOARDING_SETTINGS_UPDATE` :slightly_smiling_face:

Reply:
Ironic, I seem to have copied the wrong action while saying your action is wrong 😆